test/test_pyodide.py

```diff
@@ -84,3 +78,64 @@ def test_pyodide_build(tmp_path, use_pyproject_toml):
     print("expected_wheels", expected_wheels)
 
     assert set(actual_wheels) == set(expected_wheels)
+
+
+def test_pyodide_version_incompatible(tmp_path, capfd):
+    if sys.platform == "win32":
+        pytest.skip("emsdk doesn't work correctly on Windows")
```

Comment:
I believe that emsdk works fine on windows and it's Pyodide's fault that it doesn't work.
```suggestion
        pytest.skip("pyodide-build doesn't work correctly on Windows")
```